Source/JavaScriptCore/parser/Nodes.h

```diff
@@ -2250,6 +2250,7 @@ namespace JSC {
         FunctionMode functionMode() { return m_functionMode; }
 
         int functionNameStart() const { return m_functionNameStart; }
+        unsigned functionStart() const { return m_functionStart.has_value() ? *m_functionStart : static_cast<unsigned>(m_functionKeywordStart); }
```

Comment:
You can use `m_functionStart.value_or(static_cast<unsigned>(m_functionKeywordStart))`.